README.md

```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
 
 ### In Progress
-- [ ] Toxcore
-    - [ ] 100% unit testing
-    - [ ] Make ToxAV stateless
-    - [ ] Allow a single toxcore instance to handle multiple keypairs (or 'clients')
-    - [ ] Consistent naming scheme throughout toxcore
-    - [X] Make toxcore stateless
-- [ ] Messenger
-    - [ ] Improve group chat implementation
-    - [ ] Improve A/V implementation
-    - [ ] Multiple device support
+- [ ] 100% unit testing
```

Comment:
It's not a roadmap though, given that tasks listed in previous disorder weren't being worked on one after another..?

Well, we can always do `sort -R` to bring back disorder.